package.nls.json

```diff
@@ -55,7 +55,7 @@
     "AWS.stepFunctions.publishWizard.stateMachineNameToUpdate.noStateMachines.detail": "Create a state machine before proceeding. See documentation for details.",
     "AWS.stepFunctions.publishWizard.stateMachineNameToUpdate.title": "Select state machine to update ({0})",
     "AWS.stepFunctions.render": "Render graph",
-    "AWS.command.samVisualize.renderTemplate": "Render CloudFormation template",
+    "AWS.command.samVisualize.renderTemplate": "Render SAM Template graph",
```

Comment:
more customers are familiar with CloudFormation than SAM, this has been raised in conversations with customers. Does the feature not visualize non-SAM cloudformation templates?

Reply:
It does visualize CloudFormation templates. I'll change it back for familiarity.